osgar/zmqrouter.py

```diff
@@ -64,6 +64,9 @@ def record(config, log_prefix=None, log_filename=None, duration_sec=None):
                 duration = datetime.timedelta(seconds=duration_sec) if duration_sec else datetime.timedelta.max
                 router.run(duration=duration)
             except Exception as e:
+                (etype, evalue, tb) = sys.exc_info()
```

Comment:
I expected that you can get traceback from the exception `e` ... OK, you are right:
https://docs.python.org/3/library/traceback.html

Reply:
I expected that as well but apparently, that is not how it works.